src/JuliusSweetland.OptiKey.Core/Models/XmlKeyboardModels/XmlMultiStepKey.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) 2019 OPTIKEY LTD (UK company number 11854839) - All Rights Reserved
+using JuliusSweetland.OptiKey.Enums;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace JuliusSweetland.OptiKey.Models
+{
+    public class XmlMultiStepKey : XmlKey
```

Comment:
In order to free up the name "DynamicKey" for my new code, I renamed the old version to "MultiStepKey". It would be ideal to deprecate "MultiStepKey" when the new more advanced "DynamicKey" is released.

Reply:
Ok that sounds good to me.